### ISSUE TITLE
devspece dev with custom manifests does not work with kubectl 1.24.0

### ISSUE THREAD

**Issue by rreckel** [loft-sh/devspace#2069]
**What happened?**  
`devspace dev` fails with error: 
```
[fatal]  error deploying: error deploying manifests: malformed version: 1.24.0
Kustomize Version: v4.5.4
Please make sure `kubectl apply` does work locally with manifest `manifests.yaml`
```

Using `kubectl apply -f manifests.yaml` works without an error.

**What did you expect to happen instead?**  
No error

**How can we reproduce the bug?** (as minimally and precisely as possible)  
Use a devspace.yaml with:
```
deployments:
- name: manifests
  kubectl:
    manifests:
    - manifests.yaml
```
and install kubectl 1.24.0

**Local Environment:**  
- DevSpace Version: devspace version 5.18.4
- Operating System: mac
- ARCH of the OS: i386
**Kubernetes Cluster:**  
- Kubernetes Version: [use `kubectl version`]
Client Version: v1.24.0
Kustomize Version: v4.5.4
Server Version: v1.21.12

**Anything else we need to know?**  

The problem seems to be a change in version reportinng of kubectl.
I suppose that kubectl reports the version without a leading "v".
This fails the regex in line:
https://github.com/loft-sh/devspace/blob/1ff15ddcb76e45e1b0188e1933b689f9f4becee4/pkg/util/constraint/version.go#L63

/kind bug

**Comment by FabianKramm** [comment]
@rreckel thanks a lot for the PR! We'll release a fix for this very soon. Meanwhile you could try the new v6 alpha version, that should have a fix for this already included.

**Comment by FabianKramm** [comment]
@rreckel should be fixed with DevSpace v5.18.5

**Comment by rreckel** [comment]
It works like a charm again.
Thank you for the very quick fix!
